src/libraries/System.Numerics.Tensors/tests/TensorPrimitivesTests.cs

```diff
@@ -3149,7 +3149,7 @@ public void Subtract_TensorScalar_ThrowsForOverlapppingInputsWithOutputs()
         public void Sum_AllLengths()
         {
             // TODO https://github.com/dotnet/runtime/issues/98861
-            T? tolerance = Helpers.DetermineTolerance<T>(doubleTolerance: 1e-13, floatTolerance: 1e-5f);
+            T? tolerance = Helpers.DetermineTolerance<T>(doubleTolerance: 1e-14, floatTolerance: 1e-5f);
```

Comment:
New tolerance failing in some CI legs
```suggestion
            T? tolerance = Helpers.DetermineTolerance<T>(doubleTolerance: 1e-13, floatTolerance: 1e-5f);
```